install/mysql/glpi-empty.sql

```diff
@@ -7825,6 +7825,7 @@ CREATE TABLE `glpi_users` (
   `password_history` longtext,
   `is_notif_enable_default` tinyint DEFAULT NULL,
   `show_search_form` tinyint DEFAULT 0,
+  `search_pagination_on_top` tinyint DEFAULT 0,
```

Comment:
Default value for users should be `null`. Indeed, this `null` value will permit to fallback to the value configured globally in the `glpi_configs` table.

```suggestion
  `search_pagination_on_top` tinyint DEFAULT NULL,
```

By the way, it looks like the `show_search_form` default value should be fixed.